tests/languages/java-file/app/FileReadSample.java

```diff
@@ -0,0 +1,19 @@
+import java.io.*;
+import java.nio.file.*;
+import java.util.stream.*;
+import java.nio.charset.*;
+
+public class FileReadSample {
+	public  String readFile(String filePath) {
+	StringBuilder contentBuilder = new StringBuilder();
+		try (Stream<String> stream = Files.lines( Paths.get(filePath), StandardCharsets.UTF_8)) 
```

Comment:
I would remove the try-catch and let it crash if something goes wrong.

Reply:
Done